File: python/src/publisher/publisher_int.py
#!/usr/bin/env python3

import rospy
from std_msgs.msg import Int32

def publisher_int():
    pub = rospy.Publisher('chatter/int', Int32, queue_size=10)
    rospy.init_node('publisher_int', anonymous=True)
    rate = rospy.Rate(10) # 10hz
    while not rospy.is_shutdown():
        hello_int = 10
        rospy.loginfo(hello_int)
        pub.publish(hello_int)
        rate.sleep()

if __name__ == '__main__':
    try:
        publisher_int()
    except rospy.ROSInterruptException:
        pass

File: ros_noetic/cpp/src/publisher/publisher_int.cpp
#include "ros/ros.h"
#include "std_msgs/Int64.h"

int main(int argc, char **argv)
{
  ros::init(argc, argv, "publisher_int");
  ros::NodeHandle n;

  ros::Publisher chatter_pub = n.advertise<std_msgs::Int64>("chatter/int", 1000);

  ros::Rate loop_rate(10);
 
  while (ros::ok())
  {
    std_msgs::Int64 msg;
    msg.data = 10;

    ROS_INFO("%i", msg.data);

    chatter_pub.publish(msg);

    ros::spinOnce();

    loop_rate.sleep();
  }
  
  return 0;
}
